beesdoo_shift/models/res_partner.py

```diff
@@ -66,6 +68,24 @@ class ResPartner(models.Model):
     subscribed_shift_ids = fields.Many2many(
         comodel_name="beesdoo.shift.template", readonly=True
     )
+    shift_shift_ids = fields.One2many(
+        comodel_name="beesdoo.shift.shift",
+        inverse_name="worker_id",
+        string="Shifts",
+        help="All the shifts the worker is subscribed to.",
+    )
```

Comment:
Should this not be computed? Where is this set?

Reply:
One2many fields are computed by Odoo based on the Many2one fields in the co-model through the "inverse_name" field.